src/core_atmosphere/physics/mpas_atmphys_control.F

```diff
@@ -200,10 +200,11 @@ subroutine physics_namelist_check(configs)
  endif
 
 !convection scheme:
- if(.not. (config_convection_scheme .eq. 'off'              .or. &
-           config_convection_scheme .eq. 'cu_grell_freitas' .or. &
-           config_convection_scheme .eq. 'cu_kain_fritsch'  .or. &
-           config_convection_scheme .eq. 'cu_tiedtke'       .or. &
+ if(.not. (config_convection_scheme .eq. 'off'                 .or. &
```

Comment:
I know it's unsatisfying from a spacing perspective, but only the cu_grell_freitas line should be changed.